hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/protocolPB/ClientNamenodeProtocolTranslatorPB.java

```diff
@@ -545,16 +545,22 @@ public LocatedBlock getAdditionalDatanode(String src, long fileId,
         .newBuilder()
         .setSrc(src)
         .setFileId(fileId)
-        .setBlk(PBHelperClient.convert(blk))
```

Comment:
Can we add a test?

Reply:
thanks for your feedback, I will add a test soon.

Reply:
Thanks @hangc0276, any progress here?

Reply:
I am adding the test, and will push the code soon.